DoSomething/rogue
# History modal bug

#### What's this PR do?
- Fixes history modal so you can update the quantity on the post when `DS_ENABLE_V3_QUANTITY_SUPPORT` is set to `true`. **This only works for when this is set to `true` and will break when it is set to `false`.** 
- Disables history events log table for now (until we refactor events system)
- Disables showing the quantity at the top of the signup page until we switch to support for quantity on the post and revisit what this looks like (quantity is now visible next to each post)
- Only allows admins to update the quantity if they enter a number (button only becomes disabled if a number is entered):
![feb-21-2018 15-33-54](https://user-images.githubusercontent.com/9019452/36503880-a919274e-171c-11e8-8800-ea0c9b9372b3.gif)

#### How should this be reviewed?
1. Set the `DS_ENABLE_V3_QUANTITY_SUPPORT` to be `true`. 
2. Visit `/campaign/:nid` 
3. Update a post's quantity. Make sure the change is reflected in the database. 
4. Visit `/campaign/:nid/inbox`
5. Update a post's quantity. Make sure the change is reflected in the database. 
6. Visit `/signups/:sid`. Make sure there is no quantity at the top.
7. Try to update quantity with non-integer. This should not let you submit the form.
#### Relevant tickets
Fixes [this](https://www.pivotaltracker.com/n/projects/2019429/stories/154347334) pivotal card.

#### Checklist
